string name
string jointName
float64 qW
float64 qX
float64 qY
float64 qZ
bool offset

